cves/CVE-2019-14235.yml

```diff
@@ -115,10 +100,12 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: true
+  code_answer: |
+    The originial code was tested. The vulnerability is a util method so it was tested through the method uri_to_iri unit test.
+  fix: true
+  fix_answer: |
+    The fix added a unit test that directly tests repercent_broken_unicode by triggering the recursion error that occurs
```

Comment:
What is repercent_broken_unicode in this context? Is it the method causing the vulnerability?